src/index.html

```diff
@@ -5,8 +5,362 @@
+              <a class="nav_link" href="#">
+                SAMSUNG
+              </a>
+            </li>
+            <li class="nav_item">
+              <a class="nav_link" href="#">
+                SMARTPHONES
+              </a>
+            </li>
+            <li class="nav_item">
+              <a class="nav_link nav_link--color_blue"             data-qa="nav-hover"
+              href="#">
+                LAPTOPS & COMPUTERS
+              </a>
+            </li>
+            <li class="nav_item">
+              <a class="nav_link" href="#">
+                GADGETS
+              </a>
+            </li>
+            <li class="nav_item">
+              <a class="nav_link" href="#">
+                TABLETS
+              </a>
+            </li>
+            <li class="nav_item">
+              <a class="nav_link" href="#">
+                PHOTO
+              </a>
+            </li>
+            <li class="nav_item">
+              <a class="nav_link link" href="#">
+                VIDEO
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <main class="card">
+        <div class="card__fist-row">
+          <section class="card__col">
+            <div class="card__first" data-qa="card">
+              <img
+                class="card__image"
+                src="/images/imac.jpeg"
+                 alt="APPLE A1419 iMac27"
+              >
+              <div class="card__list">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </div>
+              <div class="card__code">Product code: 195434</div>
+              <div class="card__rate">
+                <div class="stars">
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star"></div>
+                  <div class="stars__star stars__star--last-star">
+                  </div>
+                </div>
+                <p class="card__reviews">Reviews: 5</p>
+              </div>
+
+              <div class="card__price">
+                <p class="card__price-price">
+                  Price:
+                </p>
+                <p class="card__price-number">$2,199</p>
+              </div>
+              <div class="card__buy">
+                <a href="#buy" class="card__buy-now"
+                  data-qa="card-hover">
+                    BUY
+                </a>
+              </div>
+            </div>
+          </section>
+          <section class="card__col">
+            <div class="card__second">
+              <img
+               class="card__image"
+               src="/images/imac.jpeg"
+               alt="APPLE A1419 iMac27"
+              >
+              <div class="card__list">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </div>
+              <div class="card__code">Product code: 195434</div>
```

Comment:
Do not use div for text content. Use semantic tags instead <h2> - for heading <p> - for other text